ui/src/api/api-fetch.ts

```diff
@@ -242,7 +242,7 @@ async function fetchNonNullAdt<Input, Output>(
     const result = await fetchNullableAdt<Input, Output>(path, method, body);
 
     if (result.isOk() && result.value === undefined) {
-        Snackbar.error('Expected non-null result but got null', `${method} ${path}`);
+        Snackbar.error('Expected non-null result but got null');
```

Comment:
Näiltä puuttuu käännökset, mutta suoraan sanottuna en keksi näille mitään järkevää suomennosta. Oli käännös mikä tahansa, niin melkein lukisin tämän mielummin englanniksi.

Reply:
Hmm. "Pyyntö ei palauttanut odotettua arvoa", "Pyynnön palauttama arvo ei ollut oletettu/oli odottomaton", "Konkreettista arvoa ei palautettu","Pyyntö palautti tyhjän arvon" tms.

Sinänsä epämielekästä jos jokin minimaalinen subsetti virheistä on englanniksi, vaikka se vaikuttaisikin (ainakin devaajien puolesta) järkevämmältä.

Reply:
Tuossa on useampikin fiksu ehdotus, voisin pyöritellä näitä hetken aikaa päässäni ja valkata jonkun niistä 👍 Onhan se joo järkevää että virheilmoitus on suomeksi, vaikka nämä yleisesti ottaen lienevätkin sellaisia virheitä, joita lähinnä me nähdään devatessa (ainakin toivottavasti)

Reply:
Ei välttämättä ole tarvetta käännökselle koska kyseessä on nimenomaan koodarille tarkoitettu viesti. 
Eli pyydetään non-nullia bäkkäristä, mutta jostain syystä sieltä tuleekin null --> joko bäkkäri palauttaa väärän tai frontissa on pyydetty väärällä kutsulla (meillä on fetchNonNullin lisäksi myös fetchNull)

Reply:
Joo eihän näitä oikeasti saisi tulla käyttäjälle asti mutta aina voi käydä moka. Siinä kohtaa on tosiaan parempi että se on yhtenäinen. Kyllä me se tuosta suomenkielisestäkin tajutaan että mistä oli kyse.